main.tmux

```diff
@@ -21,8 +21,15 @@ tmux set-option -g message-style "$TMUX_POWERLINE_STATUS_STYLE"
 tmux set-option -g status-left-length $TMUX_POWERLINE_STATUS_LEFT_LENGTH
 tmux set-option -g status-right-length $TMUX_POWERLINE_STATUS_RIGHT_LENGTH
 
-tmux set-option -g status-left "#(${TMUX_POWERLINE_DIR_HOME}/powerline.sh left)"
-tmux set-option -g status-right "#(${TMUX_POWERLINE_DIR_HOME}/powerline.sh right)"
+if [ "$TMUX_POWERLINE_STATUS_VISIBILITY" = "2" ]; then
+  tmux set-option -g status-format[$TMUX_POWERLINE_WINDOW_STATUS_LINE] "#[align=centre]#{W:#{E:window-status-format} ,#{E:window-status-current-format} }"
+  tmux set-option -g status-format[$TMUX_POWERLINE_OTHER_STATUS_LINE] "#[align=left]#(${TMUX_POWERLINE_DIR_HOME}/powerline.sh left)"
+  tmux set-option -ag status-format[$TMUX_POWERLINE_OTHER_STATUS_LINE] "#[align=right]#(${TMUX_POWERLINE_DIR_HOME}/powerline.sh right)"
+elif [ "$TMUX_POWERLINE_STATUS_VISIBILITY" = "on" ]; then
```

Comment:
Could we move this to be the first check if the if-else, as this is the most common case? That will make the code easier to understand.

Reply:
Sure, done.